flake.lock

```diff
@@ -0,0 +1,427 @@
+{
```

Comment:
Overall OK with having a `flake.lock`, but we really should have Renovate update this :thinking:

Reply:
It supports it now: https://github.com/renovatebot/renovate/issues/8000

But I don't remember if it updates all the inputs or only `nixpkgs` inputs.

Reply:
Right, can we somehow enable it in our config? Or is it the default behavior? :thinking: 

In that thread, I [see](https://github.com/renovatebot/renovate/issues/8000#issuecomment-1312289641)

```json
"nix": {
  "enabled": true
},
```

Reply:
Apparently, it is updating all the inputs! That's great!

https://github.com/loophp/php-src-nix/pull/52